server/app.test.js

```diff
@@ -0,0 +1,27 @@
+jest.mock('../database/db.js');
+
+const request = require('supertest');
+const app = require('./app.js');
+
+describe('GET /restaurants/:restaurantId/reviews', () => {
+  test('it should respond with 200 for existing restaurant', async () => {
```

Comment:
Cool implementation of async/await!  If you wanted to just use regular syntax, jest allows you to pass an argument called "done" into the test which you can invoke after your asynchronous call.  You probably already know this, but the documentation is here: 

https://jestjs.io/docs/en/asynchronous